web/crux/src/app/product/product.http.controller.ts

```diff
@@ -52,6 +54,7 @@ export default class ProductHttpController {
 
   @Get(ROUTE_PRODUCT_ID)
   @ApiOkResponse({ type: ProductDetailsDto })
+  @Params('productId')
```

Comment:
Let's talk about this tomorrow, cause we have some issues with the swagger, and i think it's partially. related.

Reply:
Hmm, okey.